app/controllers/api/ruby_sg_bot_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Api::RubySgBotController < Api::ApplicationController
+  def webhook
+    if ruby_sg_bot_joined_group?
+      if RubySgBotSubscriber.find_or_create_by(chat_id: tg_chat_id.to_s)
+        telegram_bot_api.send_message(
+          tg_chat_id,
+          <<~TELEGRAM_MESSAGE
+          👋 At your service!
+          TELEGRAM_MESSAGE
+        )
+      end
+    elsif ruby_sg_bot_left_group?
+      RubySgBotSubscriber.find_by(chat_id: tg_chat_id.to_s)&.destroy
+    end
+
+    head :no_content
+
+  rescue ArgumentError
+    # if return code is not 2XX, telegram api will keep sending the request.
+    #   There is no error logging for the application for the moment therefore
+    #   just letting it fail silently for now.
+    head :no_content
+  end
+
+  private
+
+  def telegram_bot_api
+    TelegramBotApi.new(ENV["RUBY_SG_BOT_TOKEN"])
+  end
+
+  def rubysg_bot_id
+    "1292813591"
+  end
+
+  def ruby_sg_bot_joined_group?
```

Comment:
I would probably extract these methods into its own Class. 
Instead of having all these methods in this controller

Reply:
Leaving it here for now because this method is highly dependent on the `#tg_object` method. Even if we extract `#tg_object` out together, it will only be used in this controller. Therefore should extract in the future when there is another use case.